pkg/daemon/ceph/osd/volume.go

```diff
@@ -79,9 +80,26 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 		strconv.Itoa(a.storeConfig.OSDsPerDevice),
 	}...)
 
-	// ceph-volume is soon implementing a parameter to specify the "fast devices", which correspond to the "metadataDevice" from the
-	// crd spec. After that is implemented, we can implement this. In the meantime, we fall back to use rook's partitioning.
+	if a.storeConfig.StoreType == config.Bluestore && a.storeConfig.DatabaseSizeMB > 0 {
+		if a.storeConfig.DatabaseSizeMB < 1024 {
```

Comment:
Can we use a const for `1024` instead?

Reply:
Sure, do you have a suggested const name for that?

Reply:
`cephVolumeMinDBSize`?

Reply:
Done.